products/firefox_iOS/testing_iOS.md

```diff
@@ -0,0 +1,22 @@
+# Testing on iOS for a new release
+We currently rely on screenshots for testing. These are provided on the [dev.l10n mailing list](https://lists.mozilla.org/listinfo/dev-l10n), which you should follow if you are working on l10n for any of the existing Mozilla products (otherwise you are missing crucial information and updates).
```

Comment:
Let's keep avoiding "we". Also, avoid repeating the same information about screenshots you provide later. e.g.

> It’s currently possible to test Firefox for iOS only by reviewing screenshots generated by the iOS team. Information and updates are provided in the [dev.l10n mailing list](https://lists.mozilla.org/listinfo/dev-l10n), which you should follow if you are working on l10n for any of the existing Mozilla products (otherwise you are missing crucial information and updates).